src/formats/wkt/Wkt.js

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (C) 2017 United States Government as represented by the Administrator of the
+ * National Aeronautics and Space Administration. All Rights Reserved.
+ */
+define([
+    '../../error/ArgumentError',
+    '../../util/Logger',
+    './WktTokens'
+], function (ArgumentError,
+             Logger,
+             WktTokens) {
+    /**
+     * Wkt is capable of parsing the text representation of the WKT objects. The explanation of what all is
+     * supported is to be found in the README.MD in this directory.<br/>
+     * <br/>
+     * The simplest possible usage is:<br/>
+     * var layer = new WorldWind.RenderableLayer();<br/>
+     * var parser = new Wkt('POINT (19 23)');<br/>
+     * parser.load(null, null, layer);<br/>
+     * wwd.addLayer(layer);<br/>
+     * This example adds the WKT into the map<br/>
+     *
+     * The more complex usage allows you to do whatever you want with the parsed objects. For example you can make
+     * sure that only points are displayed<br/>
+     * var layer = new WorldWind.RenderableLayer();<br/>
+     * var parser = new Wkt('POINT (19 23)');<br/>
+     * parser.load(function(wkt, objects){<br/>
```

Comment:
As discussed this morning, we still need to remove the objects argument of the completion callback to be fully aligned with Shapefile. We will consider this kind of additions at a later stage.